specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.4/TextAnalytics.json

```diff
@@ -0,0 +1,2448 @@
+        },
+        "entities": {
+          "description": "Healthcare entities.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/HealthcareEntity"
+          }
+        },
+        "relations": {
+          "type": "array",
+          "description": "Healthcare entity relations.",
+          "items": {
+            "$ref": "#/definitions/HealthcareRelation"
+          }
+        },
+        "warnings": {
+          "type": "array",
+          "description": "Warnings encountered while processing document.",
+          "items": {
+            "$ref": "#/definitions/TextAnalyticsWarning"
+          }
+        },
+        "statistics": {
+          "description": "if showStats=true was specified in the request this field will contain information about the document payload.",
+          "$ref": "#/definitions/DocumentStatistics"
+        }
+      }
+    },
+    "HealthcareEntity": {
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Entity"
+        },
+        {
+          "properties": {
+            "assertion": {
+              "type": "object",
+              "$ref": "#/definitions/HealthcareAssertion"
+            },
+            "name": {
+              "description": "Preferred name for the entity. Example: 'histologically' would have a 'name' of 'histologic'.",
+              "type": "string"
+            },
+            "links": {
+              "description": "Entity references in known data sources.",
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/HealthcareEntityLink"
+              }
+            }
+          }
+        }
+      ]
+    },
+    "HealthcareRelation": {
+      "type": "object",
+      "description": "Every relation is an entity graph of a certain entityType, where all entities are connected and have specific roles within the relation context.",
+      "required": [
+        "relationType",
+        "entities"
+      ],
+      "properties": {
+        "relationType": {
+          "description": "Type of relation. Examples include: `DosageOfMedication` or 'FrequencyOfMedication', etc.",
+          "type": "string"
+        },
+        "entities": {
+          "description": "The entities in the relation.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/HealthcareRelationEntity"
+          }
+        }
+      }
+    },
+    "HealthcareAssertion": {
+      "type": "object",
+      "properties": {
+        "conditionality": {
+          "description": "Describes any conditionality on the entity.",
+          "type": "string",
+          "enum": [
+            "Hypothetical",
+            "Conditional"
+          ],
+          "x-ms-enum": {
+            "name": "Conditionality",
+            "modelAsString": false
+          }
+        },
+        "certainty": {
+          "description": "Describes the entities certainty and sentiment.",
```

Comment:
Use `assertion` or `polarity` instead of `sentiment`. Sentiment has a "good" / "bad" connotation in Text Analytics.